plugin/pkg/scheduler/api/types.go

```diff
@@ -81,7 +81,7 @@ type PredicateArgument struct {
 	LabelsPresence *LabelsPresence
 }
 
-// Represents the arguments that the different types of priorities take.
+// PriorityArgument represents the arguments that the different types of priorities take.
```

Comment:
PriorityArgument represents the arguments to configure priority functions in scheduler policy configuration.